### ISSUE TITLE
Add support for Local Machine store

### ISSUE THREAD

**Issue by ifedotov-re** [mongodb-js/win-export-certificate-and-key#1]
Hi!

I noticed that [here](https://github.com/mongodb-js/win-export-certificate-and-key/blob/c97b4c81cce1a737d03f3954b7038b211c3906ab/binding.cc#L93) the CertOpenSystemStoreA function is used.

Does it mean that win-export-certificate-and-key and mongosh as a whole do not support certificates from the Local Machine store?  
See https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-certopensystemstorea :  
```
Remarks
Only current user certificates are accessible using this method, not the local machine store.
```

Whereas MongoDB checks both Local Machine and Current User stores:
https://github.com/mongodb/mongo/blob/7a68603aed1dd0b75caf0faae3a9986886af0f18/src/mongo/util/net/ssl_manager_windows.cpp#L1196

If so, could you add support for certificates from the Local Machine store?

**Comment by addaleax** [comment]
@ifedotov-re Hello, and thanks for reporting this! What you say makes sense, and I’ve opened https://jira.mongodb.org/browse/MONGOSH-1112 to track this on our side. If you want, you can also take a stab at fixing this yourself.

> Does it mean that win-export-certificate-and-key and mongosh as a whole do not support certificates from the Local Machine store?

I guess, yes. That’s not an intentional omission, though.

**Comment by addaleax** [comment]
@ifedotov-re We have released mongosh 1.2.2 with support for this, please feel free to try it out and let us know if everything works as it should!
